### PR TITLE
Reintroduce import React rules

### PR DESCRIPTION
**Pull request by mattcompiles**

### CODE REVIEW

File: template/src/render.tsx [hunk 1]
```diff
@@ -12,7 +13,7 @@ const skuRender: Render<RenderContext> = {
   renderApp: ({ SkuProvider, environment }) => {
     const appHtml = ReactDOM.renderToString(
       <SkuProvider>
-        <App environment={environment} />
+        <App environment={environment as ClientContext['environment']} />
```
Comment (mattcompiles) [inline template/src/render.tsx:16]: This is an tiny unrelated change that fixes a type error in the init template.

**Review by michaeltaranto — APPROVED** [review]

**Review by markdalgleish — APPROVED** [review]
